src/mmw/js/src/core/models.js

```diff
@@ -6,6 +6,20 @@ var Backbone = require('../../shim/backbone'),
     turfArea = require('turf-area'),
     utils = require('./utils');
 
+var handleNullInCallback = function(callbackFn) {
+    return _.wrap(callbackFn, function(fn, args) {
```

Comment:
Using `_.wrap` may be unnecessary since you can just return the wrapped function directly.

Ex.

```
return function() {
    callbackFn.apply(this, arguments);  
};
```